pfcli/deployment.py

```diff
@@ -154,10 +154,9 @@
 
 @app.command()
 def list(
-    show_all: bool = typer.Option(
+    include_terminated: bool = typer.Option(
         False,
-        "--all",
-        "-a",
+        "--include-terminated",
```

Comment:
Do we need to support list of only archived deployments?

Reply:
I do not think so. Maybe later we can support list by deployment status.